.gitignore

```diff
@@ -1,4 +1,5 @@
 .direnv/*
 .envrc
 /target
-shell.nix
\ No newline at end of file
+/wasm/build
```

Comment:
```
wasm/build/*.wasm
wasm/build/*.map
```

@gterzian can you also delete the files get_string.wasm, ".map, caller.wasm & ".map to untrack these files?

Reply:
ok, done